compiler/lists.nim

```diff
@@ -7,30 +7,35 @@
 #    distribution, for details about the copyright.
 #
 
-# This module implements a generic doubled linked list.
-# TODO Remove this and replace it with something sensible
+# This module is deprecated, don't use it.
```

Comment:
Well I did, but then there were complaints from @Araq that I can't just remove stuff. When he is ok, I would be very happy, to completely remove the ``compiler/lists.nim`` file entirely. It also has a conflicting name with the lists package of the standard library.